File: examples/wagipython/wagi-python/code/env.py
import sys
import os

print('Content-Type: text/plain; charset=UTF-8')
print('Status: 200')
print()

print('Hello from python on', sys.platform)

print()
print('### Arguments ###')
print()

print(sys.argv)

print()
print('### Env Vars ###')
print()

for k, v in sorted(os.environ.items()):
    print(k+':', v)

print()
print('### Files ###')
print()

for f in os.listdir('/code'):
    print(f)

import logging
logging.warn('Logging important warnings')

File: examples/wagiruby/wagi-ruby/lib/env.rb
puts "Content-Type: text/plain; charset=UTF-8"
puts "Status: 200"
puts "X-Foo-Header: Bar"

puts 
puts
puts "Hello from ruby!"
puts
puts "ruby version: #{RUBY_VERSION} (#{RUBY_RELEASE_DATE}) [#{RUBY_PLATFORM}]"

puts

puts "### Arguments ###"
puts
ARGV.each_with_index { |e, i| puts "arg #{i}: #{e}" }

puts

puts "### Env Vars ###"
puts
ENV.each { |k, v| puts "#{k}=#{v}" }

puts

puts "### Files ###"
puts
Dir.foreach("/") { |x| puts x }
